js/background.js

```diff
@@ -68,3 +68,183 @@ chrome.omnibox.onInputEntered.addListener(function (text) {
 	});
 });
 
+chrome.runtime.onMessage.addListener(
+	function (request, sender) {
+		if (request.msg === "myaccount_clicked" || request.msg === "promote_clicked") {
+			var datacenter = request.data.datacenter;
+			var ticketID = request.data.ticketID;
+			console.log("message received");
+			chrome.storage.sync.get("eid", function (obj) {
+				var userID = obj.eid;
+				$(document).ready(function () {
+					getUserList(datacenter, userID, request.msg, ticketID);
+				});
+			});
+		};
+	});
+
+
+function makePersonalUrl(dc, qualtricsID, ticketID) {
+	if (dc == "CO1") {
+		chrome.tabs.create({ "url": "https://odo.corp.qualtrics.com/?a=ResearchSuite&b=RSUserAccountAccess&bid=qus1&uid=" + qualtricsID + "&RSLogin&FromTicket=" + ticketID + "&Direct" });
+	}
+	else if (dc === "AZ1") {
+		chrome.tabs.create({ "url": "https://odo.corp.qualtrics.com/?a=ResearchSuite&b=RSUserAccountAccess&bid=qaz1&uid=" + qualtricsID + "&RSLogin&FromTicket=" + ticketID + "&Direct" });
+	}
+	else if (dc === "CA1") {
+		chrome.tabs.create({ "url": "https://odo.corp.qualtrics.com/?a=ResearchSuite&b=RSUserAccountAccess&bid=qcorp&uid=" + qualtricsID + "&RSLogin&FromTicket=" + ticketID + "&Direct" });
+	}
+	else if (dc === "EU1") {
+		chrome.tabs.create({ "url": "https://odo.corp.qualtrics.com/?a=ResearchSuite&b=RSUserAccountAccess&bid=qeurope&uid=" + qualtricsID + "&RSLogin&FromTicket=" + ticketID + "&Direct" });
+	}
+	else if (dc === "AU1") {
+		chrome.tabs.create({ "url": "https://odo.corp.qualtrics.com/?a=ResearchSuite&b=RSUserAccountAccess&bid=qasia&uid=" + qualtricsID + "&RSLogin&FromTicket=" + ticketID + "&Direct" });
+	}
+}
+
+function getUserList(datacenter, userID, msg, ticketID) {
```

Comment:
You should not be making requests like this. You have now exposed these tokens to all of github and to the client side since the extension runs client side.

Reply:
You need to change all these tokens within the user accounts now